### DISCUSSION TITLE
Checkpoint for MPI-INF-3DHP

### DISCUSSION THREAD

**Discussion by smahmood00** [open-mmlab/mmpose#1460] [Q&A] [answered]
Where can I get the checkpoint file for 3D baseline model on MPI-3DHP dataset?

**Comment by ly015** [comment, marked as answer]
Please check https://github.com/open-mmlab/mmpose/blob/master/configs/body/3d_kpt_sview_rgb_img/pose_lift/mpi_inf_3dhp/simplebaseline3d_mpi-inf-3dhp.md

**Reply by smahmood00** [reply]
Thanks!
